platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationContextImpl.java

```diff
@@ -196,6 +196,47 @@ public boolean cleanDirectory(Path path) {
     return true;
   }
 
+  /**
+   * Temporary version of cleanDirectory() that doesn't delete any directories but only files found.
```

Comment:
👍 Great comment! Do we have a ticket for a more permanent solution? If so, could we add it to the comment?

Reply:
Same ticket different PR. This is just a first quick solution.